Plots/Contours/NCL_polar_1_lg.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
For **docstring** (i.e. descriptive lines of codes in the beginning of the script, which is from Line 1 to Line 7 in this script), please use the format from previously implemented functions and modify it accordingly to `NCL_polar_1`. This is because we are using **Sphinx Python documentation generator** to populate our **ReadTheDocs** links with nice-looking styles.